apps/web/components/team/screens/Team.tsx

```diff
@@ -1,27 +1,24 @@
 import Link from "next/link";
-import { useSearchParams } from "next/navigation";
 
 import { useLocale } from "@calcom/lib/hooks/useLocale";
+import { useRouterQuery } from "@calcom/lib/hooks/useRouterQuery";
 import { md } from "@calcom/lib/markdownIt";
 import { markdownToSafeHTML } from "@calcom/lib/markdownToSafeHTML";
 import type { TeamWithMembers } from "@calcom/lib/server/queries/teams";
 import { Avatar } from "@calcom/ui";
 
 type TeamType = Omit<NonNullable<TeamWithMembers>, "inviteToken">;
 type MembersType = TeamType["members"];
-type MemberType = MembersType[number] & {
-  safeBio: string | null;
-};
-
-type TeamTypeWithSafeHtml = Omit<TeamType, "members"> & {
-  members: MemberType[];
-};
+type MemberType = MembersType[number] & { safeBio: string | null };
+type TeamTypeWithSafeHtml = Omit<TeamType, "members" | "inviteToken"> & { members: MemberType[] };
 
 const Member = ({ member, teamName }: { member: MemberType; teamName: string | null }) => {
-  const searchParams = useSearchParams();
+  const routerQuery = useRouterQuery();
```

Comment:
i ran [next router codemod](vscode://intuita.intuita-vscode-extension/showCodemod?chd=-wqkAQr7ILgYeTRozWTEgiUvmSY) on this file and got no change. we need to improve this codemod. cc @DmytroHryshyn @grzpab